Dockerfile

```diff
@@ -1,55 +1,40 @@
-##
-#Copyright 2014 Cisco Systems Inc. All rights reserved.
```

Comment:
Any reason copyright is removed?

Reply:
Because i think it's not useful given there's `LICENSE` file under this repo already